### PR TITLE
SD-124: Update drone version 1

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
**Jira Ticket**: https://collaboration.homeoffice.gov.uk/jira/browse/SD-124

**Changes**: Updated drone.yaml file to version 1

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,124 +1,151 @@
...
-  build_brpapp:
-    image: docker:18.03.1-ce
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-    commands:
-      - docker build -t app-$${DRONE_COMMIT_SHA} .
-    when:
-      event: pull_request
+steps:
+- name: build_10
```
Comment (sasdevelopers) [inline .drone.yml:7]: Can we name these pipeline steps similarly in all of the pipelines

File: .drone.yml [hunk 2]
```diff
@@ -1,124 +1,151 @@
...
+  commands:
+    - cd kube-brpapp
+    - bin/deploy.sh
+  environment:
+    KUBE_SERVER_PROD:
+      from_secret: kube_server_prod
+    KUBE_TOKEN_PROD:
+      from_secret: kube_token_prod
+    KUBE_NAMESPACE: brp
+    APP_IMAGE_TAG: ${DRONE_COMMIT_SHA}
```
Comment (sasdevelopers) [inline .drone.yml:135]: Whats this used for?  Althoug maybe its in the deployment files in whihc case leave it alone

**Review by sasdevelopers — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (SarahJaneLuff) [reply .drone.yml]: changed the step to build, but it is not present in ARE

File: .drone.yml (hunk 2 above)
Comment (SarahJaneLuff) [reply .drone.yml:149]: KUBE_NAMESPACE is used the the deployment.yaml file, and APP_IMAGE_TAG has been changed to "BRPAPP_IMAGE_TAG", and is present in app-deployment.yaml.

**Pull request merged**